packages/cli-repl/test/e2e.spec.ts

```diff
@@ -1143,5 +1142,120 @@ describe('e2e', function() {
       shell.assertContainsOutput('610');
     });
   });
+
+  describe('external editor', () => {
+    let db;
+    let client;
+    let shell: TestShell;
+    let dbName;
+    let homedir: string;
+    let env: Record<string, string>;
+
+    beforeEach(async() => {
+      const homeInfo = setTemporaryHomeDirectory();
+      const connectionString = await testServer.connectionString();
+
+      dbName = `test-${Date.now()}`;
+      homedir = homeInfo.homedir;
+      env = homeInfo.env;
+
+      shell = TestShell.start({
+        args: [ connectionString ],
+        forceTerminal: true,
+        env
+      });
+
+      client = await MongoClient.connect(connectionString, {});
+      db = client.db(dbName);
```

Comment:
Do we need `dbName`, `db`, `client`, `connectionString` etc.? I’m not seeing any database calls here (and I don’t think we need any, either), so this could be a test that uses `--nodb`